src/main/java/edu/kis/vh/nursery/HanoiRhymer.java

```diff
@@ -2,16 +2,25 @@
 
 public class HanoiRhymer extends defaultCountingOutRhymer {
```

Comment:
defaultCountingOutRhymer should start from Uppercase. Problems with merging 'format'?